Packs/PrismaCloud/ReleaseNotes/4_3_1.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Prisma Cloud v2
+
+Fixed an issue where the ***test module*** failed on Docker timeout when the url is not reachable.
```

Comment:
```suggestion
Fixed an issue where the ***test module*** failed on container timeout when server url is not reachable.
```

Reply:
I also changed from `test module` to `test button` (validate BA-125)